pcsx2/DEV9/ConfigUI.cpp

```diff
@@ -184,16 +184,16 @@ class DEV9Dialog : public wxDialog
 		hdd_grid->Add(m_hdd_file,     wxSizerFlags().Expand());
 		auto* hdd_size_label = new wxStaticText(this, wxID_ANY, _("HDD Size (GiB):"));
 		auto* hdd_size_box = new wxBoxSizer(wxHORIZONTAL);
-		m_hdd_size_slider = new wxSlider(this, wxID_ANY, HDD_MIN_GB, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_MIN_MAX_LABELS | wxSL_AUTOTICKS);
+		m_hdd_size_slider = new wxSlider(this, wxID_ANY, 10, HDD_MIN_GB, HDD_MAX_GB, wxDefaultPosition, wxDefaultSize, wxSL_HORIZONTAL | wxSL_MIN_MAX_LABELS | wxSL_AUTOTICKS);
 		m_hdd_size_slider->SetPageSize(10);
-		for (int i = 15; i < HDD_MAX_GB; i += 5)
-			m_hdd_size_slider->SetTick(i);
+		for (int i = HDD_MIN_GB; i < HDD_MAX_GB; i += 5)
+			m_hdd_size_slider->SetTickFreq(10);
```

Comment:
you shouldn't have to do this in a for-loop.  It looks like the original method was manually setting the ticks -- but the whole point of TickFrequency is specifying "hey i want you to put a tick every nth increment"